charts/consul/test/unit/server-acl-init-job.bats

```diff
@@ -633,14 +633,14 @@ load _helpers
   [ "${actual}" = "${expected}" ]
 
   # Check that the bootstrap token flag is set to the path of the Vault secret.
-  local actual=$(echo $object | jq -r '.spec.containers[] | select(.name="post-install-job").command | any(contains("-bootstrap-token-file=/vault/secrets/bootstrap-token"))')
+  local actual=$(echo $object | jq -r '.spec.containers[] | select(.name=="post-install-job").command | any(contains("-bootstrap-token-file=/vault/secrets/bootstrap-token"))')
```

Comment:
fixing a bug with `jq`. It didn't matter because there was only one container so it was selecting that one